lib/rubocop/cop/rails/file_path.rb

```diff
@@ -7,6 +7,9 @@ module Rails
       # to use `Rails.root.join` clause. It is used to add uniformity when
       # joining paths.
       #
+      # `Rails.root.join` is not a drop-in replacement. It returns a `Pathname`
+      # object which may require further changes to your code.
```

Comment:
I think a code example will help users understand. For example, the PR description helped me understand.